src/app/contract/contractParser.js

```diff
@@ -128,7 +88,7 @@ var formatGasEstimates = function (data) {
   var ret = {}
   var fun
   if ('creation' in data) {
-    ret['Creation'] = gasToText(data.creation[0]) + ' + ' + gasToText(data.creation[1]) + '\n'
```

Comment:
Hm wondering why this even had a new line?

Reply:
I think before that block was displayed in a big <pre>